ansible/roles/common/tasks/metrics.yml

```diff
@@ -33,7 +33,7 @@
         state: absent
 
   when: |
-    node_exporter_capture.rc != 0 or not node_exporter_capture.stderr.startswith(
+    node_exporter_capture.rc != 0 or not node_exporter_capture.stdout.startswith(
```

Comment:
FYI: this is a small bug fix. The version gets printed to stdout and not stderr. Without this change, the block was always being run.